vendor/modules.txt

```diff
@@ -215,7 +216,7 @@ github.com/ulikunitz/xz/internal/hash
 # github.com/vmihailenco/msgpack v4.0.1+incompatible
 github.com/vmihailenco/msgpack
 github.com/vmihailenco/msgpack/codes
-# github.com/vmware/go-vcloud-director/v2 v2.2.0
+# github.com/vmware/go-vcloud-director/v2 v2.3.0-alpha.1
```

Comment:
The tag is incorrect as we're now working on v2.4.0. I know it's easy to miss, given that we released v2.3.0 so quickly :)

Reply:
govcd tags a little diff story now: https://github.com/vmware/go-vcloud-director/tags

Reply:
All right, let's officially decouple the versions of these two projects and let them live their own lives (versions). And, hopefully, this will lead to many new users of go-vcloud-director! :)